taxonomies/ingredients.txt

```diff
@@ -240,6 +239,7 @@ stopwords:zh::配料
 
 
 fr:additifs alimentaires
+	'dont acides gras satur(e|é)s
```

Comment:
Why is this here?

Reply:
I don't know, some bug with my script to add the translations. I fixed it manually.